UI/src/controller/trafficSignsReadOnlyCollection.js

```diff
@@ -19,10 +19,10 @@
     var trafficSignValues = {
       speedLimit: { values : [1, 2, 3, 4, 5, 6]},
       trSpeedLimits: { values : [1, 2, 3, 4, 5, 6]},
-      totalWeightLimit: {values : [32]},
-      trailerTruckWeightLimit: {values : [33]},
-      axleWeightLimit:{values : [34]},
-      bogieWeightLimit: {values : [35]},
+      totalWeightLimit: {values : [32, 33, 34, 35]},
+      trailerTruckWeightLimit: {values : [32, 33, 34, 35]},
+      axleWeightLimit:{values : [32, 33, 34, 35]},
+      bogieWeightLimit: {values : [32, 33, 34, 35]},
```

Comment:
Once the value are the same, could be create a generic like `weightLimit: {values : [32, 33, 34, 35]}`